pandas/tests/window/test_groupby.py

```diff
@@ -926,3 +926,71 @@ def test_pairwise_methods(self, method, expected_data):
 
         expected = df.groupby("A").apply(lambda x: getattr(x.ewm(com=1.0), method)())
         tm.assert_frame_equal(result, expected)
+
+    def test_times(self):
```

Comment:
Could you split this into 3 different tests? Some duplication is okay but nice to have each test case isolated.